src/app/dashboard/executions/executions.component.html

```diff
@@ -29,7 +29,12 @@ <h3>{{ 'allexecutions' | translate }}</h3>
             {{execution.metisPlugins[execution.currentPlugin].pluginType}}
           </span>
         </td>
-        <td class="nowrap">{{execution.metisPlugins[execution.currentPlugin] ? execution.metisPlugins[execution.currentPlugin].executionProgress.processedRecords : '-'}}</td>
+        <td class="nowrap">
```

Comment:
There is duplicate code: lines 24-45 are identical to lines 67-88. The tables with plugin executions are quite similar.